packages/block-library/src/paragraph/edit.js

```diff
@@ -209,7 +223,7 @@ class ParagraphBlock extends Component {
 					>
 						<ContrastChecker
 							{ ...{
-								textColor: textColor.color,
+								textColor: this.getDisplayedTextColor( backgroundColor.class, textColor.color ),
```

Comment:
Upon inspection of the file, it seems like it might be necessary to provide defaults inside the implementation of `withFallbackStyles` at the top of the file in place of `undefined` values.